docs/docker/README.md

```diff
@@ -102,17 +114,6 @@ $ docker exec -it keymaster /usr/bin/htpasswd -B -c \
  /etc/keymaster/passfile.htpass $USERNAME
 ```
 
-## Add users
-
-By default a user is created. Let's start by deleting this and creating our own
-user.
-
-```
-$ rm -f ${KEYMASTER_DATA}/conf/passfile.htpass
-$ docker exec -it keymaster /usr/bin/htpasswd -B -c \
- /etc/keymaster/passfile.htpass $USERNAME
-```
-
```

Comment:
On the docker config... is the htpassword file not used?

Reply:
It is. I deleted this section because my original doc had a copy/paste error. This section is there twice.